src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -108,20 +108,24 @@ final class OnboardingInfoViewController: UIViewController {
 		if pageType == .togetherAgainstCoronaPage { loadCountryList() }
 	}
 
+	
 	override func viewWillAppear(_ animated: Bool) {
 		super.viewWillAppear(animated)
 
 		let preconditions = exposureManager.preconditions()
 		updateUI(exposureManagerState: preconditions)
 	}
 
+	
 	override func viewDidLayoutSubviews() {
 		super.viewDidLayoutSubviews()
 
 		scrollView.contentInset.bottom = footerView.frame.height - scrollView.safeAreaInsets.bottom
 		scrollView.verticalScrollIndicatorInsets.bottom = scrollView.contentInset.bottom
 	}
 
+	
+	// MARK: - Public
```

Comment:
These are actually `Internal`, not `Public` by default. And maybe you can quickly check if they could be set to private or are actually used from outside the class :)

Reply:
Good point. All these funcs can be declared as private. Question: how do you sort methods in a section? Alphabetically?

Reply:
There is no strict rule, but my feeling is that I usually put methods that belong together next to each other, and often the ones that are called earlier are more at the top. But I guess everyone has their own preferences there. I think the order should somehow make sense and not just be alphabetically.